fundingmanager.go

```diff
@@ -2559,7 +2558,6 @@ func (f *fundingManager) handleInitFundingMsg(msg *initFundingMsg) {
 		msg.err <- err
 		return
 	}
-	addMaxValueFlag
```

Comment:
this belongs to the fundingmanager commit

Reply:
Sorry about that. 

fixed.